AcademyApp/iOSNights2019/Scenes/Lectures/LecturesViewController.swift

```diff
@@ -35,7 +35,7 @@ class LecturesViewController: UIViewController {
 private extension LecturesViewController {
     func setupUI() {
         tableView.register(LectureCell.nib, forCellReuseIdentifier: LectureCell.reuseIdentifer)
-
+        tableView.register(UINib(nibName: "LectureHeaderView", bundle: nil), forHeaderFooterViewReuseIdentifier: "Header")
```

Comment:
Try to define and use LectureHeaderView.nib instead as in the case of LectureCell.nib